tools/nightly_build_and_test_stats.groovy

```diff
@@ -32,17 +32,18 @@ node ("master") {
   def testStats = []
 
   stage("getPipelineStatus") {
-    if (variant == "temurin") {
-        variant = "hotspot"
+    def apiVariant = variant
```

Comment:
Can we make this `trssVariant` or `trssApiVariant` instead of `apiVariant` to disambiguiate it from the main adoptium API?

Reply:
it is the adoptium API..
```${apiUrl}/v3/assets```

Reply:
the problem was this code (for the publish health status) was overriding the variant variable to hotspot...

Reply:
@sxa

Reply:
Thanks for the clarification. LGTM